scripts/pylib/twister/twisterlib/runner.py

```diff
@@ -218,7 +218,7 @@ def run_build(self, args=[]):
                     log.write(log_msg)
 
             if log_msg:
-                overflow_found = re.findall("region `(FLASH|ROM|RAM|ICCM|DCCM|SRAM)' overflowed by", log_msg)
+                overflow_found = re.findall("region `(FLASH|ROM|RAM|ICCM|DCCM|SRAM|dram0_1_seg)' overflowed by", log_msg)
```

Comment:
This feels somewhat hacky ... any reason to not generalise this such that it skips for any memory region overflows other than those that are not supposed to overflow (can only think of `IDT_LIST`)?

Reply:
@stephanosio because I'm just trying to get the esp32 tests to be handled correctly as easily as I can.  I leave improves to twister itself to the twister developers :).

Reply:
Fair enough.